ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-vpws.md

```diff
@@ -101,11 +101,11 @@ interface Management1
 
 ### Router BGP VPWS Instances
 
-| Instance | Route-Distinguisher | Both Route-Target | Pseudowire | Local ID | Remote ID |
-| -------- | ------------------- | ----------------- | ---------- | -------- | --------- |
-| TENANT_A | 100.70.0.2:1000 | 65000:1000 | TEN_A_site1_site3_pw | 15 | 35 |
-| TENANT_A | 100.70.0.2:1000 | 65000:1000 | TEN_A_site2_site5_pw | 25 | 57 |
-| TENANT_B | 100.70.0.2:2000 | 65000:2000 | TEN_B_site2_site5_pw | 26 | 58 |
+| Instance | Route-Distinguisher | Both Route-Target | MPLS Control Word | Label Flow | MTU | Pseudowire | Local ID | Remote ID |
+| -------- | ------------------- | ----------------- | ----------------- | -----------| --- | ---------- | -------- | --------- |
+| TENANT_A | 100.70.0.2:1000 | 65000:1000 | True | True | 1601 | TEN_A_site1_site3_pw | 15 | 35 |
+| TENANT_A | 100.70.0.2:1000 | 65000:1000 | True | True | 1601 | TEN_A_site2_site5_pw | 25 | 57 |
+| TENANT_B | 100.70.0.2:2000 | 65000:2000 | False | False | ignore | TEN_B_site2_site5_pw | 26 | 58 |
```

Comment:
This test will fail. Need to update artifacts

Reply:
Was too quick on the trigger, running molecule now...